bindings/dart/ffigen_common.yaml

```diff
@@ -6,6 +6,3 @@ headers:
     - 'ios/Classes/XaynAiFfiCommon.h'
   include-directives:
     - 'ios/Classes/XaynAiFfiCommon.h'
-compiler-opts-automatic:
```

Comment:
Is this ok now? @rustonaut We may need to change this in the discovery engine

Reply:
compilation fails if set to `false` now, so i had to set it to `true`. the generated output seems to be the same. i'm not sure if this was needed before (maybe to mirror the settings in `cbindgen`?) and i also don't know why it worked before and not anymore.